app/views/kaminari/_paginator.html.erb

```diff
@@ -0,0 +1,23 @@
+<%# The container tag
+  - available local variables
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+    paginator:     the paginator that renders the pagination tags inside
+-%>
+<%= paginator.render do -%>
```

Comment:
What does `-%>` do?

Reply:
Ah that's just an alternate way of closing. Hadn't noticed, as it was autogenerated

Reply:
@kicferk1  few more tricks on this link: https://stackoverflow.com/questions/7996695/what-is-the-difference-between-and-in-erb-in-rails

Reply:
I was just looking into this 🙂 No newline

Reply:
I'd got confused with `<%- -%>`